src/JBMultiTerminal.sol

```diff
@@ -2018,7 +2020,7 @@ contract JBMultiTerminal is JBPermissioned, ERC2771Context, IJBMultiTerminal {
         // slither-disable-next-line reentrancy-events
         netAmountPaidOut = amountPaidOut
             - (
-                _isFeeless(_msgSender())
+                _isFeeless(owner)
```

Comment:
I wonder what address we should be checking here, we could instead also check if the `beneficiary` if feeless :thinking:

Reply:
yeah, could be beneficiary. i think its best as owner. could be both. who knows.